rpc-jobs/sync-credential-store.yml

```diff
@@ -0,0 +1,68 @@
+- job:
+    name: 'Sync-Credential-Store'
+    project-type: workflow
+    concurrent: false
+    parameters:
+      - rpc_gating_params
+      - string:
+          name: "PWSAFE_PROJECT_ID"
+          description: |
+            PasswordSafe Project ID
+      - password:
+          name: "SSO_USERNAME"
+          description: |
+            SSO username to connect to PasswordSafe with
+      - password:
+          name: "SSO_PASSWORD"
+          description: |
+            SSO password to connect to PasswordSafe with
+      - password:
+          name: "PIP_PWSAFE_LOCATION"
+          description: |
+            pip location to install pwsafe library from,
+            eg. git+ssh://git@repo_url/user/pwsafe.git
+
+    dsl: |
+      node(){ // CIT slave node
+        withCredentials([
+          file(
+            credentialsId: 'service_account_jenkins_ssh_key',
+            variable: 'JENKINS_SSH_KEY'
+          )
+        ]){
+          dir("rpc-gating"){
+            git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
+            dir("scripts"){
+              TMP_DIR = pwd(tmp:true)
+              withEnv([
+                "TMP_DIR=${TMP_DIR}",
+                "PWSAFE_PROJECT_ID=${PWSAFE_PROJECT_ID}",
+                "JENKINS_URL=${JENKINS_URL}",
+                "JENKINS_SSH_KEY=${JENKINS_SSH_KEY}",
+                "SSO_USERNAME=${SSO_USERNAME}",
+                "SSO_PASSWORD=${SSO_PASSWORD}"])
+              {
+                sshagent (credentials: ['rpc-jenkins-svc-github-key']) {
+                  sh """#!/bin/bash
+                    scl enable python27 bash
```

Comment:
Does this work in a non-interactive context? is it equivalent to ` source /opt/rh/python27/enable`?